src/services/identity/__tests__/SitesService.spec.ts

```diff
@@ -863,7 +865,7 @@ describe("SitesService", () => {
       )
 
       // Assert
-      expect(actual).toBeInstanceOf(NotFoundError)
+      expect(actual).toEqual(err(new MissingSiteError()))
```

Comment:
nit: should we be using `toBeInstanceOf(NotFoundError)` as in the future if we wish to add metadata to the error object, this assert will fail?